### ISSUE TITLE
Spoon version 3.7.0 not working with overriding entry types any more

### ISSUE THREAD

**Issue by peirix** [angell-co/Spoon#126]
I set up a project last year with Spoon 3.7.0. There I had set up some overrides for each entry type on which matrix blocks should be available. Opening that project now I can no longer find anywhere to change these override settings. I'm pretty sure that was something I could do from Sections -> SectionName -> Fields -> Field settings.

<img width="297" alt="image" src="https://github.com/doublesecretagency/craft-spoon/assets/347175/437a875b-ba38-4ca4-a4bb-105c4a84490f">

Somewhere in there, right?

The settings still apply as seen here:
<img width="119" alt="image" src="https://github.com/doublesecretagency/craft-spoon/assets/347175/3f05e418-5499-4f3f-9018-15b6d209f0e9">

But not everywhere, as you can see here (all blocks are available using the cogwheel on an existing block):
<img width="240" alt="image" src="https://github.com/doublesecretagency/craft-spoon/assets/347175/54c1d6ef-95d1-42de-a7f5-6893928ab9c8">

**Comment by samuelreichor** [comment]
Same problem here :(

**Comment by samuelreichor** [comment]
@peirix Any updates?

**Comment by peirix** [comment]
Nope. Guessing they've stopped working on the Craft 3 version of the plugin 🤷

**Comment by lindseydiloreto** [comment]
Hey, sorry for the late reply.

Spoon was overhauled significantly in v4.0 (for Craft 4), so it's tricky to go back to the old codebase (which I did not originally write). I'm guessing that something changed within Craft itself, during some Craft 3.x update.

Which specific version of Craft 3 are you running? And is there any chance of you updating to Craft 4 anytime soon? It's worth noting that Craft 3 itself is [almost unsupported](https://craftcms.com/knowledge-base/supported-versions).

This issue is technically on our radar, but yes, it's a low priority. If either of you would care to submit a PR for it (on the `v3` branch), we would happily accept it.

**Comment by samuelreichor** [comment]
@lindseydiloreto I have the Issue with Craft 4.4.16.1 and Spoon Version 4.0.4 :((

**Comment by lindseydiloreto** [comment]
> I have the Issue with Craft 4.4.16.1 and Spoon Version 4.0.4

Oh. Well that's a different situation entirely.

Let me take a look (at Spoon 4 for Craft 4) and let you know what I find.

**Comment by lindseydiloreto** [comment]
Hi @samuelreichor, can you please double-check your version numbers, and see if you can still recreate this behavior? I just tested locally with **Craft v4.4.16.1** and **Spoon v4.0.4**, and everything worked normally.

![Ω  2023-08-22  12 34 18](https://github.com/doublesecretagency/craft-spoon/assets/5309692/82266540-ea19-4af9-938d-79337c6d46cd)

If you are still experiencing this issue (with Craft/Spoon 4), please provide some additional information about where you are getting stuck.

**Comment by samuelreichor** [comment]
Hi again @lindseydiloreto, I am doing exactly this steps but nothing changes. Spoon's configuration files are written correctly as far as I can see, but that doesn't change the fact that certain fields are hidden at the entry type level.

Unfortunately, I can't find any further information in the logs (Do Spoon logs exist at all?).

In the meantime, I have also tested it on a Linux server and got the same results.
Let me know if you need more

**Comment by lindseydiloreto** [comment]
Hi @samuelreichor, I'm not sure what you may be bumping into. DM me on Discord (@lindseydiloreto) and we can troubleshoot the issue together in real-time.

Whatever we discover, we'll report back here on this thread.
